{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"failure"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"Codable","preciseIdentifier":"s:s7Codablea"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/openweathersdk\/apierror\/errormodel\/failure(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.daksh.OpenWeatherSDK\/documentation\/OpenWeatherSDK\/APIError\/ErrorModel\/failure(_:)","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"A case that encapsulates the failure response data as a Codable object."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"failure"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"Codable","preciseIdentifier":"s:s7Codablea"},{"kind":"text","text":")"}],"title":"APIError.ErrorModel.failure(_:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:14OpenWeatherSDK8APIErrorO10ErrorModelO7failureyAESe_SEpcAEmF","modules":[{"name":"OpenWeatherSDK"}]},"hierarchy":{"paths":[["doc:\/\/com.daksh.OpenWeatherSDK\/documentation\/OpenWeatherSDK","doc:\/\/com.daksh.OpenWeatherSDK\/documentation\/OpenWeatherSDK\/APIError","doc:\/\/com.daksh.OpenWeatherSDK\/documentation\/OpenWeatherSDK\/APIError\/ErrorModel"]]},"references":{"doc://com.daksh.OpenWeatherSDK/documentation/OpenWeatherSDK/APIError/ErrorModel":{"role":"symbol","title":"APIError.ErrorModel","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ErrorModel"}],"abstract":[{"type":"text","text":"A model representing a failure response in the API."}],"identifier":"doc:\/\/com.daksh.OpenWeatherSDK\/documentation\/OpenWeatherSDK\/APIError\/ErrorModel","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ErrorModel"}],"url":"\/documentation\/openweathersdk\/apierror\/errormodel"},"doc://com.daksh.OpenWeatherSDK/documentation/OpenWeatherSDK":{"role":"collection","title":"OpenWeatherSDK","abstract":[],"identifier":"doc:\/\/com.daksh.OpenWeatherSDK\/documentation\/OpenWeatherSDK","kind":"symbol","type":"topic","url":"\/documentation\/openweathersdk"},"doc://com.daksh.OpenWeatherSDK/documentation/OpenWeatherSDK/APIError":{"role":"symbol","title":"APIError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"APIError"}],"abstract":[{"type":"codeVoice","code":"APIError"},{"type":"text","text":" is the error type returned. It encompasses a few different types of errors, each with"},{"type":"text","text":" "},{"type":"text","text":"their own associated reasons."}],"identifier":"doc:\/\/com.daksh.OpenWeatherSDK\/documentation\/OpenWeatherSDK\/APIError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"APIError"}],"url":"\/documentation\/openweathersdk\/apierror"},"doc://com.daksh.OpenWeatherSDK/documentation/OpenWeatherSDK/APIError/ErrorModel/failure(_:)":{"role":"symbol","title":"APIError.ErrorModel.failure(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"failure"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"Codable","preciseIdentifier":"s:s7Codablea"},{"kind":"text","text":")"}],"abstract":[{"type":"text","text":"A case that encapsulates the failure response data as a Codable object."}],"identifier":"doc:\/\/com.daksh.OpenWeatherSDK\/documentation\/OpenWeatherSDK\/APIError\/ErrorModel\/failure(_:)","kind":"symbol","type":"topic","url":"\/documentation\/openweathersdk\/apierror\/errormodel\/failure(_:)"}}}